lib/endpoints/class-wp-json-controller.php

```diff
@@ -196,4 +200,111 @@ public function get_endpoint_args_for_item_schema( $add_required_flag = true ) {
 
 		return $post_type_fields_args;
 	}
+
+	/**
+	 * Validate an parameter value that's based on a property from the item schema.
+	 * 
+	 * @param  mixed $value
+	 * @param  WP_JSON_Request $request
+	 * @param  string $parameter
+	 * @return WP_Error|bool
+	 */
+	public function validate_schema_property( $value, $request, $parameter ) {
```

Comment:
It would be good to have a strong set of tests for this `validate_schema_property()` and the `sanitize_schema_property()` methods, independently of how they're used in our controllers.